.gitignore

```diff
@@ -15,3 +15,5 @@ symbulation.test
 .Rproj.user
 *.gcno
 *.gcda
+signalgp-lite

```

Comment:
We don't want signalgp-lite ignored, so please remove it. If you need it in your local .gitignore, you should just make sure not to add and push this file